### DISCUSSION TITLE
Zarla hangisi kazanır sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#312] [Zeka Sorusu]
Damla ve Selin bir zar oyunu oynarlar. İki zarı atacaklardır, 8 gelirse Selin 9 gelirse Damla kazanacaktır. Oyuna Damla başladığına göre hangisinin kazanma olasılığı daha fazladır?

**Comment by slos8888** [comment]
**Selinin kazanma oranı daha fazladır.**
Selinin kazanma oranı daha fazladır. Çünkü 8 ile 3 kombinasyon(2,6/3,5/4,4) yapılabilirken 9 ile sadece iki kombinasyon yapılabilir(3,6/4,5). Bu yüzden Selin'in kazanma oranı daha fazladır.
